boefjes/boefjes/plugins/kat_maxmind_geoip/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Arguments",
+  "type": "object",
+  "properties": {
+    "MAXMIND_USER_ID": {
+      "title": "User ID",
+      "type": "string",
+      "maxLength": 2048,
```

Comment:
User ID is numeric for MaxMind Api, and probably a postive integer.

Reply:
Both are 40 or 41 chars. Lets limit them to 48 to be sure. (i could not find a spec on them)